users/common/utils.py

```diff
@@ -18,23 +21,28 @@ def generate_random_phone_number(length):
 
 
 def raise_serializer_exception(error_message):
+    logger.warning(f"Serializer exception with message: {error_message}")
     raise serializers.ValidationError(error_message)
 
 
 def raise_custom_validation_exception(error_message, error_code):
+    logger.warning(f"Custom valid exception with message: {error_message} and error code: {error_code}")
     raise CustomValidationError(error_message, error_code)
 
 
 def prepare_proper_exception(call_instance, error_message, error_code):
     if "serializers" in str(type(call_instance)):
+        logger.debug(f"Preparing exception caught in serializer instance: {call_instance}")
```

Comment:
I don't really get what means `Preparing exception caught...`. Maybe better would be: ```Serializer: {call_instance} caught exception with message: {error_message} and error_code: {error_code}```? Or maybe even delete this log?

Reply:
I will change it as you want, if in the end you decide to delete it I will do it